services/spiget/spiget-download-size.service.js

```diff
@@ -3,9 +3,7 @@
 const { BaseSpigetService, documentation, keywords } = require('./spiget-base')
 
 module.exports = class SpigetDownloadSize extends BaseSpigetService {
-  static get category() {
-    return 'size'
-  }
+  static category = 'size'
 
   static get route() {
```

Comment:
@RDIL Convert this also to static property

Reply:
Whoops, missed that.